### PR TITLE
Complete basic voter flow, closes #6.

### LINKED ISSUE #6: Voting Booth Views

### PR DESCRIPTION
**Pull request by bportnoy**

### CODE REVIEW

File: client/VotingBooth.html [hunk 1]
```diff
@@ -0,0 +1,12 @@
+<!doctype html>
+<html lang="en">
+  <head>
+    <meta charset="utf-8">
+    <title>Voting Booth</title>
+  </head>
+  <body>
+    <div id="election"></div>
+    <script src="JSXTransformer.js"></script>
```
Comment (ryyann) [inline client/VotingBooth.html]: I could be wrong, but doesn't grunt precompile for us already?

**Comment by ryyann** [comment]
Looks good, but to be honest I wouldn't know what looks bad at this point with my react understanding.

File: client/VotingBooth.html (hunk 1 above)
Comment (bportnoy) [inline client/VotingBooth.html]: Thanks - removed this from the new PR I'm submitting.

**Comment by bportnoy** [comment]
Closing in favor of another branch - git issues.

**Pull request closed without merging**